tileserv/tools/sfms_generator.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date, timedelta
+from typing import List
+
+
+def generate_actual(issue_date: date, for_date: date):
+    """
+        Actuals can be recorded multiple times,
+        e.g actual for 24 on 24, but also actual for 23 on 24.
+
+        It is possible that _more_ stations are reporting actuals
+        on "follow up" actuals captured days later.
+    """
+
+    """
+    for_date remains unchanged
+    issue_date will have the 5pm time attached to the date
+
+   """
+
+
+def generate_forecast(issue_date: date, for_date: date):
+    """ 
+        Forecasts will be generated multiple times for any given date.
+
+        Generally 3 to 4 forecasts will be generated for a single date
+        since that's what we've seen SFMS output
+
+        e.g. issue_date: 24
+                    - forecast: 25
```

Comment:
Aside - is SFMS not running using morning forecast data for the date of interest? I.e., for issue_date=24, there should also be a forecast for 24?

Reply:
No idea, that's why I left these as comments and put it up for an early review. I suspect you know more than I do.

Reply:
Looking at the data, yes it looks like what you stated is the case